src/languages/swift.js

```diff
@@ -123,45 +161,139 @@ export default function(hljs) {
       SINGLE_LINE_STRING(),
       SINGLE_LINE_STRING("#"),
       SINGLE_LINE_STRING("##"),
-      SINGLE_LINE_STRING("###"),
+      SINGLE_LINE_STRING("###")
     ]
   };
+
+  // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID412
+  const QUOTED_IDENTIFIER = {
+    begin: concat(/`/, Swift.identifier, /`/)
+  };
+  const IMPLICIT_PARAMETER = {
+    className: 'variable',
+    begin: /\$\d+/
+  };
+  const PROPERTY_WRAPPER_PROJECTION = {
+    className: 'variable',
+    begin: `\\$${Swift.identifierCharacter}+`
+  };
+  const IDENTIFIERS = [
+    QUOTED_IDENTIFIER,
+    IMPLICIT_PARAMETER,
+    PROPERTY_WRAPPER_PROJECTION
+  ];
+
+  // https://docs.swift.org/swift-book/ReferenceManual/Attributes.html
+  const AVAILABLE_ATTRIBUTE = {
+    begin: lookahead(/(@|#)available/), // also matches #available
+    relevance: 0,
+    contains: [
+      {
+        className: 'keyword',
+        begin: /(@|#)available/
+      },
+      {
+        begin: /\(/,
+        end: /\)/,
+        keywords: Swift.availabilityKeywords.join(' '),
+        contains: [
+          OPERATOR,
+          NUMBER,
+          STRING
+        ]
+      }
+    ]
+  };
+  const KEYWORD_ATTRIBUTE = {
+    className: 'keyword',
+    begin: concat(/@/, either(...Swift.keywordAttributes))
+  };
+  const USER_DEFINED_ATTRIBUTE = {
+    className: 'meta',
+    begin: concat(/@/, Swift.identifier)
+  };
+  const ATTRIBUTES = [
+    AVAILABLE_ATTRIBUTE,
+    KEYWORD_ATTRIBUTE,
+    USER_DEFINED_ATTRIBUTE
+  ];
+
+  const TYPE = {
+    className: 'type',
+    begin: '\\b[A-Z][\\w\u00C0-\u02B8\']*',
```

Comment:
Please convert to regex.

Reply:
I've already done that in the code for the next PR, as I'll have to change this rule anyway to fix https://github.com/highlightjs/highlight.js/issues/2895